ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.hbs

```diff
@@ -11,9 +11,9 @@
  *
  **/
  --}}
-<button class="filter-rearrange">
+<div class="filter-rearrange">
```

Comment:
We're also going to need to do the same thing [here](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/component/filter-builder/filter-builder.hbs#L15-L17)